### PR TITLE
[Incubator/elasticsearch] Allow alternate schedulers

### PR DESCRIPTION
**Pull request by knorby**
**What this PR does / why we need it**: Adds ability to use alternative schedulers, like stork

### CODE REVIEW

File: incubator/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 0.4.9
+version: 0.4.10
```
Comment (timstoop) [inline incubator/elasticsearch/Chart.yaml]: You're adding a new feature, so please bump the minor.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by knorby** [comment]
Sorry! I noticed a big bug here after submitting that I somehow didn't catch before. It should be good now.

**Comment by timstoop** [comment]
From the errors, I'm guessing that discovery is broken (again). Please be aware that there's currently a full rewrite under way for this Chart, which (among lots of other things) changes the way discovery works as the current one is not very reliable, it seems.

If you're really bent on getting this PR accepted anyway, you may consider changing the default `rbac.enable` to `true` instead, which I think will fix the e2e error.

Pinging @rendhalver @mikn as they're doing most of the rewrite. You guys might want to see if this PR can be incorporated as well, as I think it's a useful one. Due to ES storage requirements, it makes sense to use non-default, more specialised StorageClasses. IMHO, of course.

**Comment by timstoop** [comment]
PR #3074 is btw the main one for the rewrite, I think.

**Comment by rendhalver** [comment]
All the ES fixes are in #2889.
As you would expect the elastic-stack chart is basically a wrapper around elasticsearch/kibana/logstash/curator and probably fluentd when I get the chance.

**Comment by knorby** [comment]
It doesn't seem like RBAC fixes the discovery issues on the e2e test. I'm not anxious to get this merged, but I do want to make sure there are good options on storage on the rewrite, including the scheduler obviously.

**Comment by rendhalver** [comment]
@knorby The DNS discovery fixes got merged a while ago. Do you want to make another pass at this or rebase this PR?

**Comment by knorby** [comment]
@rendhalver thanks for the reminder, I hadn't been tracking the state of the chart! It is now rebased and should be fine. This is still a really small change, and the bugs from discovery/RBAC weren't related anyway, so it should be fine now.

**Comment by davidkarlsen** [comment]
@knorby can you please sign off your commits? TIA

**Pull request closed without merging**